tests/driver_srf02/main.c

```diff
@@ -55,10 +72,46 @@ int main(void)
     else {
         puts("[Ok]\n");
     }
+    return 0;
+}
 
-    while(1) {
-        xtimer_usleep_until(&wakeup, SAMPLE_PERIOD);
-        uint16_t distance = srf02_get_distance(&dev, TEST_MODE);
-        printf("distance = %i cm\n", distance);
+static int cmd_sample(int argc, char **argv)
+{
+    (void)argc;
+    (void)argv;
+    sample_loop();
+
+    return 0;
+}
+
+static int cmd_set_addr(int argc, char **argv)
+{
+    uint8_t new_addr;
+
+    if (argc < 2) {
+        printf("usage: %s <new_addr (decimal)>\n", argv[0]);
```

Comment:
In the header file you changed the format of `SRF02_DEFAULT_ADDR` to hex but here you ask for decimal numbers. I'd like to unify that